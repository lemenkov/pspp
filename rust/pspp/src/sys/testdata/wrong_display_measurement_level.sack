# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 1; 1; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Numeric variable, no label or missing values.
2; 0; 0; 0; 0x050800 *2; s8 "NUM1";

# Display parameters record.
7; 11; 4; 2; >>4<<; 0;

# Character encoding record.
7; 20; 1; 12; "windows-1252";

# End of dictionary.
999; 0;
