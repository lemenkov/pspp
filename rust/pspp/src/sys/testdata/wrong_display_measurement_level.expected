Warning at file offsets 0xe0 to 0xe8: In variable display record: Invalid variable measurement level value 4.

╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │SAV                     │
│       Number of Cases│Unknown                 │
╰──────────────────────┴────────────────────────╯

╭─────────┬─╮
│Variables│1│
╰─────────┴─╯

                                               Variables
╭────┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│    │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├────┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│num1│       1│     │                 │Input│    8│Left     │F8.0        │F8.0        │              │
╰────┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯
